### DISCUSSION TITLE
Are extension loaded by rtp always activated

### DISCUSSION THREAD

**Discussion by GordianDziwis** [neoclide/coc.nvim#2867] [Q&A] [answered]
`CocList extension` always show an `*` in front of extension loaded with vim-plug. For example coc-jedi is prefixed with an `*` when a tex file ist opened. Is this a drawback from using vim-plug or intentional?

**Comment by fannheyward** [comment, marked as answer]
`coc-jedi` sets `activationEvents` to `*`, this makes it will be activated for all filetypes, https://github.com/pappasam/coc-jedi/blob/master/package.json#L43. Should be fixed in coc-jedi for Python only.
